apps/mobile/src/libs/helpers/create-radio-button-properties/create-radio-button-properties.helper.ts

```diff
@@ -0,0 +1,61 @@
+import { BaseColor } from "~/libs/enums/enums";
+import { globalStyles } from "~/libs/styles/styles";
+import {
+	type RadioButtonProps,
+	type RadioGroupItemStyle,
+	type RadioGroupOption,
+	type StyleProp,
+	type TextStyle,
+} from "~/libs/types/types";
+
+type Properties = {
+	commonStyles?: StyleProp<TextStyle>;
+	itemContainerStyle?: RadioGroupItemStyle;
+	label: React.ReactNode;
+	option: RadioGroupOption;
+	selectedId: string;
+};
+
+const CIRCLE_SIZE = 16;
+
+const BorderSize = {
+	ACTIVE: 4,
+	DEFAULT: 0,
+} as const;
+
+const createRadioButtonPropertiesHelper = ({
```

Comment:
```suggestion
const createRadioButtonProperties = ({
```
Only the file name should contain `helper`, not the variable name